spec/unit/defines/server/default_privileges_spec.rb

```diff
@@ -0,0 +1,288 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'spec_helper_acceptance'
```

Comment:
This is wrong. A unit test should not include the acceptance helper and has broken the test suite further than it already was.